lib/common.go

```diff
@@ -124,12 +124,22 @@ type (
 		NodeID      p2p.NodeID `json:"node-id"`
 		Epoch       Epoch      `json:"epoch"`
 		Initialized bool       `json:"initialized"`
```

Comment:
should the master's status be a member of the meta data?

Reply:
`Initialized` tells whether the master has ever been initialized, it is used in failover scenario only.

Reply:
As for the status of master, it has not been determined, leaving a TODO following.

Reply:
PTAL @nongfushanquan